### PR TITLE
[FEATURE] V1.0.5 Set wp_remote_post method as default to POST data to…

### PR DESCRIPTION
**Pull request by ramonfincken**

### CODE REVIEW

File: lib/ReCaptcha/ReCaptcha.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function __construct($secret, RequestMethod $requestMethod = null)
         if (!is_null($requestMethod)) {
             $this->requestMethod = $requestMethod;
         } else {
-            $this->requestMethod = new RequestMethod\Post();
+            $this->requestMethod = new RequestMethod\WP();
```
Comment (irvinlim) [inline lib/ReCaptcha/ReCaptcha.php]: Let's not modify the original library files, perhaps we can pass in the `$requestMethod` in the `ReCaptcha` constructor?

File: readme.txt [hunk 2]
```diff
@@ -15,7 +15,7 @@ This is an add-on for the [Advanced Custom Fields](http://wordpress.org/extend/p
 
 You can find more information on Google's No-CAPTCHA reCAPTCHA [here](https://www.google.com/recaptcha/intro/index.html).
 
-Requires cURL to be enabled on your server.
+This plugin does not require CuRL to be enabled on your server.
```
Comment (irvinlim) [inline readme.txt]: It should be spelt cURL.

File: acf-recaptcha-v5.php [hunk 3]
```diff
@@ -238,7 +238,7 @@ function validate_value( $valid, $value, $field, $input ){
 		if (!strlen($value)) 
 			return false;
 
-		$api = new \ReCaptcha\ReCaptcha($field['secret_key']);
+		$api = new \ReCaptcha\ReCaptcha($field['secret_key']); // FIXME : select field for request method?
```
Comment (irvinlim) [inline acf-recaptcha-v5.php]: We should pass in the `new RequestMethod\WP()` object in this constructor call.

File: README.md [hunk 4]
```diff
@@ -46,8 +46,9 @@ acf_add_local_field(array(
 
 ## About
 
-Version 1.0.4
+Version 1.0.5
 
 Written by Irvin Lim. If you encounter any issues, do open [one](https://github.com/irvinlim/acf-recaptcha/issues/new).
+WordPress wp_remote_post usage in WP Class written by Ramon Fincken (v 1.0.5)
```
Comment (irvinlim) [inline README.md]: Would appreciate it if the Markdown was written properly! Single newlines are not displayed in Markdown.

Perhaps you can create a **Contributors** section? Thanks!

File: README.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ If you haven't already, [generate your API keys first](https://www.google.com/re
 
 ### Installation and Usage
 
-This plugin requires cURL to be enabled on your server, as it makes use of Google's [PHP library](https://github.com/google/recaptcha).
+This plugin does not require CuRL to be enabled on your server
```
Comment (irvinlim) [inline README.md]: You can just remove this line actually.

**Review by irvinlim — CHANGES_REQUESTED** [review]
Hi, thank you for this! I appreciate it.

I see that you have created a class within the ReCaptcha library folder. I created this plugin 2 years ago when I still was not familiar with Composer, and included the files directly into this repository. However, for the purposes of separating WordPress code from original library code, I think that it is better if we placed this `WP.php` file somewhere else, like `includes/WPHttpRequestMethod.php`.

Anyway, still thank you for this.

**Comment by ramonfincken** [comment]
I will check on those!

File: lib/ReCaptcha/ReCaptcha.php (hunk 1 above)
Comment (ramonfincken) [reply lib/ReCaptcha/ReCaptcha.php]: Done

File: readme.txt (hunk 2 above)
Comment (ramonfincken) [reply readme.txt]: done

File: acf-recaptcha-v5.php (hunk 3 above)
Comment (ramonfincken) [reply acf-recaptcha-v5.php]: done

File: README.md (hunk 5 above)
Comment (ramonfincken) [reply README.md]: indeed, done

File: README.md (hunk 4 above)
Comment (ramonfincken) [reply README.md]: done, i didn't tag it 1.0.6, that's up to you

**Pull request closed without merging**